definitions/npm/lodash_v4.x.x/flow_v0.104.x-/lodash_v4.x.x.js

```diff
@@ -544,19 +552,17 @@ declare module "lodash" {
     // Collection
     countBy<T>(array: $ReadOnlyArray<T>, iteratee?: ?ValueOnlyIteratee<T>): { [string]: number, ... };
     countBy<T>(array: void | null, iteratee?: ?ValueOnlyIteratee<T>): {...};
-    countBy<T: Object>(object: T, iteratee?: ?ValueOnlyIteratee<T>): { [string]: number, ... };
+    countBy(string: string, iteratee?: ?ValueOnlyIteratee<string>): { [string]: number, ... };
```

Comment:
I don't think `string` is a valid type for the `collection` argument.

Reply:
You can always go to lodash.com and run lodash in the console to try it out.
It does work
e.g.
```
_.every('abc', (a) => (console.log(a), true));
```
As to whether its luck or intetentional, you'd have to ask Dalton
I agree its not documented.
Probably luck...

Reply:
Ah ok, well if the library doesn't complain then I'm happy with this change.